### ISSUE TITLE
incorrect jdaviz command line gives extraneous error that hides the real problem

### ISSUE THREAD

**Issue by eteq** [spacetelescope/jdaviz#223]
The correct way to call up specviz from the command line is:
```
jdaviz --layout=specviz /path/to/file.fits
```
However, if one accidentally leaves off the equals sign (see #224 ...), e.g.,
```
jdaviz --layout specviz /path/to/file.fits
```
instead of getting some error that the syntax couldn't be interpreted, I get a long traceback (included below), which seems to be coming from voila instead of `jdaviz`.  So we should be catching this somehow.

**Comment by nmearl** [comment]
This should now work with #226.

**Comment by javerbukh** [comment]
Fixed as of #226, closing.
